domains/client/domain-operator/src/bundle_processor.rs

```diff
@@ -243,17 +241,9 @@ where
             )
             .await?;
 
-        // TODO: Retrieve using consensus chain runtime API
-        let head_receipt_number = domain_block_result.header_number - One::one();
-        // let head_receipt_number = self
-        // .consensus_client
-        // .runtime_api()
-        // .head_receipt_number(consensus_block_hash, self.domain_id)?
-        // .into();
-
         assert!(
             domain_block_result.header_number > head_receipt_number,
-            "Consensus chain number must larger than execution chain number by at least 1"
+            "Domain chain number must larger than execution chain number by at least 1"
```

Comment:
Sorry, I don't get it. This assertion is about comparing the domain block number with the head receipt number and since the domain block is not mapping to the consensus block one by one anymore, it is weird to mention the consensus chain here IMO.

Reply:
The mapping of consensus block number and domain block number is no longer 1:1 but the assertion still remains valid. My main point is using the domain chain number and execution chain number at the same time is sort of confusing in this context as they are usually considered the same thing unless you specifically say the execution chain number refers to the receipt chain/block tree maintained on the consensus chain. The consensus chain was mentioned to not make it confusing.

Reply:
Okay, will replace execution chain with receipt chain.